src/lightning/fabric/utilities/seed.py

```diff
@@ -113,6 +118,9 @@ def _collect_rng_states(include_cuda: bool = True) -> Dict[str, Any]:
     }
     if include_cuda:
         states["torch.cuda"] = torch.cuda.get_rng_state_all()
+    if include_xpu:
+        if XPUAccelerator.is_available():
+            states["torch.xpu"] = XPUAccelerator._collect_rng_states()
```

Comment:
better to use `torch.xpu`, rather than `XPUAccelerator`.

Reply:
Yeah I was thinking the same , but would it not be better to have all abstractions of form "torch.xpu" inside the XPUAccelerator class?